README.md

```diff
@@ -15,10 +15,12 @@ To install, add the following to your project `:dependencies`:
 
 ## Usage
 
-This library currently only supports a single [Integrant][] key:
-`:duct.middleware.buddy/authentication`. This keyword wraps the
-[buddy.auth.middleware/wrap-authentication][wrap-auth] function. The
-authentication backend is denoted by a keyword on the `:backend` key:
+This library currently only supports two [Integrant][] keys:
+
+### :duct.middleware.buddy/authentication
```

Comment:
Can you put a newline after headers and wrap lines at 72 characters where possible?